regression_tests/regression_test.go

```diff
@@ -0,0 +1,131 @@
+	const adminXPriv = "xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK"
+	const actor1Alias = "Actor1RegressionTest"
+	const actor2Alias = "Actor2RegressionTest"
+
+	ctx := context.Background()
+	leaderService1, actorService1 := initTestServices(ctx, t, &services.LeaderServiceConfig{
+		EnvURL:     lookupEnvOrDefault("CLIENT_ONE_URL", ""),
+		EnvXPriv:   lookupEnvOrDefault("CLIENT_ONE_LEADER_XPRIV", ""),
+		AdminXPriv: adminXPriv,
+	}, actor1Alias)
+
+	leaderService2, actorService2 := initTestServices(ctx, t, &services.LeaderServiceConfig{
+		EnvURL:     lookupEnvOrDefault("CLIENT_TWO_URL", ""),
+		EnvXPriv:   lookupEnvOrDefault("CLIENT_TWO_LEADER_XPRIV", ""),
+		AdminXPriv: adminXPriv,
+	}, actor2Alias)
+
+	defer t.Cleanup(func() {
+		err := leaderService1.RemoveActors(ctx)
+		require.NoError(t, err, "Cleanup failed: could not actors by leader service %s: %v", leaderService1.Name())
+	})
+
+	defer t.Cleanup(func() {
+		err := leaderService2.RemoveActors(ctx)
+		require.NoError(t, err, "Cleanup failed: could not actors by leader service %s: %v", leaderService2.Name())
+	})
+
+	actor1 := actorService1.Actor()
+	actor2 := actorService2.Actor()
+
+	const minimalTransactionOutputValue = int64(-1)
+
+	// First transfer from leader2 to actor1
+	const firstTransferFunds = 3
+
+	leader2Transaction, err := leaderService2.TransferFunds(ctx, actor1.Paymail(), firstTransferFunds)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, minimalTransactionOutputValue, leader2Transaction.OutputValue)
+
+	expectedActor1Balance := uint64(1)
+	actor1Balance, err := actorService1.Balance(ctx)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, actor1Balance, expectedActor1Balance)
+
+	expectedActor1TransactionsCount := 1
+	actor1Transactions, err := actorService1.Transactions(ctx)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, len(actor1Transactions), expectedActor1TransactionsCount)
+
+	// Second transfer from leader1 to actor2
+	const secondTransferFunds = 2
+
+	leader1Transaction, err := leaderService1.TransferFunds(ctx, actor2.Paymail(), secondTransferFunds)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, minimalTransactionOutputValue, leader1Transaction.OutputValue)
+
+	expectedActor2Balance := uint64(1)
+	actor2Balance, err := actorService2.Balance(ctx)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, actor2Balance, expectedActor2Balance)
+
+	expectedActor2TransactionsCount := 1
+	actor2Transactions, err := actorService2.Transactions(ctx)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, len(actor2Transactions), expectedActor2TransactionsCount)
+
+	// Third transfer from actor1 to actor2
+	const thirdTransferFunds = 2
+
+	actor1Transaction, err := actorService1.TransferFunds(ctx, actor2.Paymail(), thirdTransferFunds)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, minimalTransactionOutputValue, actor1Transaction.OutputValue)
+
+	minimalActor1Balance := uint64(0)
```

Comment:
In other variables you name them `expectedXYZ` so I suggest to change it here aswell. 
```suggestion
	expectedActor1Balance = uint64(0)
```